samples/wifi/wfa_qt_app/README.rst

```diff
@@ -56,14 +98,67 @@ Also, run the following command to create a Serial Line Internet Protocol (SLIP)
 
 .. code-block:: console
 
-   tunslip6 -s <serial_port> <IPv6_prefix>
+   tunslip6 -s <serial_port> -T <IPv6_prefix>
+
+* ``tunslip6``: Creates a SLIP interface on the host PC, which can be used for serial communication.
+* ``serial_port``: Can be replaced with the device path on which the DUT is connected to.
+* ``IPv6_prefix``: Can be replaced with the desired IPv6 address and subnet prefix length for your device.
+
+The following is an example of the CLI command:
+
+.. code-block:: console
+
+   tunslip6 -s /dev/ttyACM4 -T 2001:db8::1/64
+
+.. note::
+   The SLIP interface can be created only on a Linux PC.
 
 See also :ref:`cmake_options` for instructions on how to provide CMake options.
 
+Sample output
+=============
+
+After programming, the sample shows the following output:
+
+.. code-block:: console
+
+   *** Booting nRF Connect SDK 48f33c9870f1 ***
+   Starting nrf7002dk_nrf5340_cpuapp with CPU frequency: 128 MHz
+   [00:00:00.330,932] <inf> net_config: Initializing network
+   [00:00:00.330,932] <inf> net_config: Waiting interface 2 (0x20007a58) to be up...
+   [00:00:00.331,085] <inf> wpa_supp: Successfully initialized wpa_supplicant
+   [00:00:00.331,237] <inf> wfa_qt: Welcome to use QuickTrack Control App DUT version v2.1
+
+   [00:00:00.331,268] <inf> wfa_qt: QuickTrack control app running at: 9004
+   [00:00:00.331,268] <inf> wfa_qt: Wireless Interface: wlan0
+   [00:00:00.757,965] <inf> net_config: Interface 1 (0x20007940) coming up
+   [00:00:00.758,087] <inf> net_config: Running dhcpv4 client...
+   [00:00:00.763,732] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000c not supported
+   [00:00:00.830,627] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
+   [00:00:00.832,214] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
+   [00:00:00.832,519] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
+   [00:00:00.834,594] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
+   [00:00:00.835,021] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
+   [00:00:00.835,327] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
+   [00:00:00.835,937] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
+
+QuickTrack certification testing
+********************************
+
+QuickTrack certification tests can be done by individual companies with Wi-Fi Alliance membership or by authorized test houses.
+See `QuickTrack Test Tool User Manual`_ for how to install the QuickTrack test tool.
```

Comment:
Links seems to be giving page not found error, are these for the registered users only?